force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +236,18 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
-
+        String helloWorld = '';
+        Integer i;
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
-
+        for (i = 0; i< repeatCount;i++) {
+            helloWorld = helloWorld + inputString;
+            if (i < (repeatCount - 1)) {
+                helloWorld = helloWorld + '; ';
```

Comment:
Updating this to use the addition assignment operator will make it a little more readable. Also, try to be more descriptive with your variable names. We don't know that the result string will be "hello world" so a better name might be `resultString` or `finalString` or `result`